portfolio-form-submission.js

```diff
@@ -66,8 +67,7 @@ const submitFunc = (event) => {
 
   if (isUserNameValid && isEmailValid && isPasswordValid) {
     submitApiRequest(userName.value, email.value, password.value)
```

Comment:
If the API gives an error?  It need to be handle

Reply:
This behaviour is already handled through catch() block on line # 80.